spotbugs/src/main/java/edu/umd/cs/findbugs/TextUICommandLine.java

```diff
@@ -278,7 +278,8 @@ public boolean justPrintVersion() {
         if (index >= 0) {
             Path path = Paths.get(optionExtraPart.substring(index + 1));
             try {
-                OutputStream oStream = Files.newOutputStream(path, StandardOpenOption.CREATE, StandardOpenOption.WRITE);
+                OutputStream oStream = Files.newOutputStream(path, StandardOpenOption.CREATE, StandardOpenOption.WRITE,
+                        StandardOpenOption.TRUNCATE_EXISTING);
```

Comment:
Alternatively, this could have been changed to use the implicity/default behavior of `newOutputStream()` by not specifying any OpenOptions:

`OutputStream oStream = Files.newOutputStream(path);`